manifests/v3.11.0/kube-apiserver/defaultconfig.yaml

```diff
@@ -1,2 +1,114 @@
 apiVersion: kubecontrolplane.config.openshift.io/v1
 kind: KubeAPIServerConfig
+admissionPluginConfig:
+  ExternalIPRanger:
+    configuration:
+      allowIngressIP: true
+      apiVersion: v1
+      externalIPNetworkCIDRs: null
+      kind: ExternalIPRangerAdmissionConfig
+    location: ""
+  openshift.io/RestrictedEndpointsAdmission:
+    configuration:
+      apiVersion: v1
+      kind: RestrictedEndpointsAdmissionConfig
+      restrictedCIDRs:
+      - 10.3.0.0/16 # ServiceCIDR
+      - 10.2.0.0/16 # ClusterCIDR
+    location: ""
+aggregatorConfig:
+  proxyClientInfo:
+    certFile: /etc/kubernetes/secrets/apiserver-proxy.crt
+    keyFile: /etc/kubernetes/secrets/apiserver-proxy.key
+apiServerArguments:
+  storage-backend:
+  - etcd3
+  storage-media-type:
+  - application/vnd.kubernetes.protobuf
+auditConfig:
+  auditFilePath: ""
+  enabled: false
+  logFormat: ""
+  maximumFileRetentionDays: 0
+  maximumFileSizeMegabytes: 0
+  maximumRetainedFiles: 0
+  policyConfiguration: null
+  policyFile: ""
+  webHookKubeConfig: ""
+  webHookMode: ""
+authConfig:
+  oauthMetadataFile: ""
+  requestHeader:
+    clientCA: /etc/kubernetes/secrets/aggregator-ca.crt
+    clientCommonNames:
+    - kube-apiserver-proxy
+    - system:openshift-aggregator
+    extraHeaderPrefixes:
+    - X-Remote-Extra-
+    groupHeaders:
+    - X-Remote-Group
+    usernameHeaders:
+    - X-Remote-User
+  webhookTokenAuthenticators: null
+consolePublicURL: ""
+corsAllowedOrigins:
+- //127\.0\.0\.1(:|$)
+- //localhost(:|$)
+imagePolicyConfig:
+  externalRegistryHostname: ""
+  internalRegistryHostname: docker-registry.default.svc:5000
+kubeletClientInfo:
+  ca: ""
+  certFile: /etc/kubernetes/secrets/apiserver.crt
+  keyFile: /etc/kubernetes/secrets/apiserver.key
+  port: 10250
+oauthConfig:
+  alwaysShowProviderSelection: false
+  assetPublicURL: # To be filled
+  grantConfig:
+    method: auto
+    serviceAccountMethod: prompt
+  identityProviders:
+  - challenge: true
+    login: true
+    mappingMethod: claim
+    name: anypassword
+    provider:
+      apiVersion: v1
+      kind: AllowAllPasswordIdentityProvider
```

Comment:
?!  Issue to remove this.  If we include it, then we have complications around overriding it.  Removing it leaves its configuration up to a higher level caller and our CI system will need to figure out how to set it.

Reply:
done